### DISCUSSION TITLE
Feature Request: Have cell bar color for a table chart be customizable.

### DISCUSSION THREAD

**Discussion by cass-simplelegal** [apache/superset#21860] [Ideas]
We would like to change the Bar color for a table chart from Gray (default) to something else, like blue or green in a Dashboard. There is no Color Scheme option under Customize tab for a "table" chart except the Conditional formatting . 

We are on Superset version 1.5 in production and I have the latest dev version locally. 

Odes anyone know if it possible to customize these colors? Tried with the label_colors , but since the colors are dynamic by amount, I couldn't quite get it to work. 

Any ideas?

Screenshot below:

<img width="1145" alt="Screen Shot 2022-10-18 at 10 56 30 AM" src="https://user-images.githubusercontent.com/116036593/196468194-05371a40-5c9a-409d-ba0b-f323170fa8d9.png">

**Comment by isaiaslafon** [comment]
We totally need this. I couldn't find the way to get that bars in CSS dashboard editor!
